storybookjs/storybook
# Add mockingbot to list of examples

Issue:

## What I did
Add MockingBot's UI library to examples.

## How to test
View examples on doc site.

- Is this testable with Jest or Chromatic screenshots?
No

- Does this need a new example in the kitchen sink apps?
No

- Does this need an update to the documentation?
Yes

If your answer is yes to any of these, please make sure to include it in your PR.
